src/modules/views/TaskView.jsx

```diff
@@ -1,37 +1,51 @@
 import React, { useContext, useState, useEffect } from "react";
 import AppContext from "../app/AppContext";
 import { makeStyles } from "@material-ui/core";
-import {PageContainer, TitleContainer, SmallContainer, MediumContainer} from "../../ui/containers/index";
+import {
+  PageContainer,
+  TitleContainer,
+  SmallContainer,
+  MediumContainer,
+} from "../../ui/containers/index";
 import { getTaskById } from "../common/apiCalls";
 
 const useStyles = makeStyles(theme => ({
-  s: {},
-}))
-
-
-const TaskView = (props) => {
+  innerContainer: {
+    'h1, h2, h3, h4, p': {
+      margin: 0,
+    },
+  },
+}));
 
+const TaskView = ({ id }) => {
   const [state, dispatch] = useContext(AppContext);
   const [error, setError] = useState(null);
-  
-  const classes = useStyles()
+  const classes = useStyles();
+  const {selectedTask: { attributes }} = state;
 
-  // useEffect(async () => {
-//   await getTaskById(id)
-//     .then(data => addTaskToState('selectedMission', data.data))
-//     .catch(error => setError(error))
-//   }, [])
+  useEffect(async () => {
+    await getTaskById(id)
+      .then(data => addTaskToState("selectedTask", data.data))
+      .catch(error => setError(error));
+  }, []);
```

Comment:
We'll also have to to fetch the tasks on the mission view for the category icons. But wondering if that will be attached to mission tasks